bundle/Resources/public/js/views/cof-contentcreationview.js

```diff
@@ -353,6 +353,10 @@ YUI.add('cof-contentcreationview', function (Y) {
                      .removeClass(CLASS_HIDDEN);
 
             createContentView.set('active', true);
+            createContentView.after('versionChange', function () {
```

Comment:
Convert this callback function into a _protected_ method of the view. Add missing description and name it accordingly in a meaningful way.